metricbeat/module/elasticsearch/node/_meta/data.json

```diff
@@ -47,4 +47,4 @@
     "service": {
         "name": "elasticsearch"
     }
-}
\ No newline at end of file
+}
```

Comment:
Nit: This change probably happened because you opened this file manually but it does not belong into this PR.

Reply:
Yeah hmm and I thought it's just an extra line but after deleting the extra line, it still shows up different.